tests/tracing/test_misc.py

```diff
@@ -374,68 +184,3 @@ def test_should_propagate_trace_to_sentry(
     client.transport.parsed_dsn = Dsn(dsn)
 
     assert should_propagate_trace(client, url) == expected_propagation_decision
-
-
-def test_start_transaction_updates_scope_name_source(sentry_init):
-    sentry_init(traces_sample_rate=1.0)
-
-    scope = sentry_sdk.get_current_scope()
-
-    with start_transaction(name="foobar", source="route"):
-        assert scope._transaction == "foobar"
-        assert scope._transaction_info == {"source": "route"}
```

Comment:
Could probably keep this, maybe not checking on the scope but just on the span directly that name and origin are set correctly.
Again, maybe this is already covered by POTel tests. In that case ignore

Reply:
if it's just on the span, it's covered elsewhere